MetricsUploader/include/MetricsUploader/MetricsUploader.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_METRICS_UPLOADER_METRICS_UPLOADER_H_
+#define ORBIT_METRICS_UPLOADER_METRICS_UPLOADER_H_
+
+#ifdef _WIN32
+#include <windows.h>
```

Comment:
It's only about HINSTANCE, isn't it? So `#include <WinDef.h>` should be enough.

Reply:
Including only `<WinDef.h>` cause `fatal error C1189: #error:  "No Target Architecture"` for me.